mmdet3d/datasets/det3d_dataset.py

```diff
@@ -58,6 +60,9 @@ class Det3DDataset(BaseDataset):
             which can be used in Evaluator. Defaults to True.
         file_client_args (dict, optional): Configuration of file client.
             Defaults to dict(backend='disk').
+        show_ins_num_var (bool, optional): Whether to show variance of the
```

Comment:
show_ins_variance:  For debug purpose. Whether to show the variance of the number of instances....